packages/components/psammead-leading-story-promo/src/index.test.jsx

```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+import { shouldMatchSnapshot } from '@bbc/psammead-test-helpers';
+import { latin } from '@bbc/gel-foundations/scripts';
+import { Headline, Summary, Link } from '@bbc/psammead-story-promo';
+import LeadingStoryPromo from './index';
+
+const Image = <img src="https://foobar.com/image.png" alt="Alt text" />;
+
+// eslint-disable-next-line react/prop-types
+const Info = () => (
+  <>
+    <Headline script={latin} service="news">
+      <Link href="https://www.bbc.co.uk/news">The headline of the promo</Link>
+    </Headline>
+    <Summary script={latin} service="news">
```

Comment:
should we also have a test for a RTL service to make sure that we don't introduce any breaking changes in the future?

Reply:
Added a snapshot test, but it mostly just ensures the right typography is used. There is no structural change in page elements. I don't think it's necessary. Thoughts?

Reply:
my thinking was more that in the future when we make adjustments to the component without checking the RTL layout, at least the snapshot test _should_ flag up inconsistencies